### DISCUSSION TITLE
D\

### DISCUSSION THREAD

**Discussion by wangdejuan** [apache/doris#19684] [Ideas]

**Comment by yagagagaga** [comment]
You can add `KRB5_CONFIG=/app/cloudera/CDH-6.3.2/krb5/krb5.conf` in your be.conf and fe.conf and then retry.

**Reply by yagagagaga** [reply]
Can you show your full error log?

**Reply by yagagagaga** [reply]
Maybe you can add `KRB_HOME=/app/cloudera/CDH-6.3.2/krb5` in your be.conf and fe.conf and then restart Doris.

**Reply by yagagagaga** [reply]
where's your krb home?
